docs/docs/getting-started/quick-tour.md

```diff
@@ -0,0 +1,57 @@
+# Quick tour
+
+This is a quick tour into the basic operations with EventStoreDb. We will look at creating a connection, writing an event and reading an event.
+
+## Requirements
+
+These examples have the following requirements:
+- At least [.NET Core SDK 3.1](https://dotnet.microsoft.com/download)
+- [Docker](https://www.docker.com/get-started)
+- A reference to the [EventStore.Client.Grpc](https://www.nuget.org/packages/EventStore.Client.Grpc/) NuGet package
+
+## Setup the certificates and running the server
+
+To run the EventStoreDB, create a new file called `docker-compose.yml` and copy the following contents into it.
+
+<<< @/samples/quick-start/docker-compose.yml
+
+Then run the command.
+
+```
+docker-compose up
+```
+
+This will launch a new instance of the EventStoreDb server .
+
+## Creating a connection
+
+Create a new console application.
+
+The following example shows the simplest way to create a connection to EventStoreDb on your local machine.
+
+<<< @/samples/quick-start/Program.cs#creating-connection
+
+EventStoreDB runs secure by default. To avoid setting up any certificates we need to override the `HttpClientHandler` to tell it to trust all certificates.
+
+:::tip
+By default, the server listens to port 2113 for requests.
```

Comment:
should this mention >1 port? is 2113 actually correct?

Reply:
gRPC is our primary protocol and all the demos are focused on it. gRPC runs over the external HTTP(S), so 2113 is correct.